agent/config/pbench-agent.cfg

```diff
@@ -6,3 +6,6 @@ pbench_web_server = pbench.example.com
 [config]
 path = %(pbench_install_dir)s/config
 files = pbench-agent-default.cfg
+
+[results]
+#server_ca =
```

Comment:
That's done by the `-default.cfg` file; the `pbench-agent.cfg` is just the overrides.

Reply:
Arguably; the separation isn't very clean, although it's slightly better than it used to be. We have the `# CHANGE ME!` comment here for the server address, and this is strongly connected ... albeit only for non-production context. I decided at the time that it crossed the line where awareness at this level was relevant. I'll re-think about it again.

Reply:
It's appropriate to have the `pbench_web_server` line in `pbench-agent.cfg`, because any installation which wants to push results needs that (and we would like to think that that is or will be all of them...).  And, until we can switch to canonical packaging (i.e., the Fedora guidlines), we need `pbench_install_dir`.  But, given that we don't need the CA in production deployments (which we expect to be the typical case), and given that the option is included in the `-defaults.cfg` file (for documentation purposes if nothing else), I don't think it's needed in the `pbench-agent.cfg` file.

But, in the grand scheme of things, I guess this is not a big deal -- it just seemed easy to address if we could land on consensus.